skbio/io/format/genbank.py

```diff
@@ -39,47 +37,113 @@
+[3]_. The feature keys and their qualifiers are also described in this
+webpage [4]_.
+
+The ``FEATURES`` section will be stored in ``interval_metadata`` of
+``Sequence`` or its sub-class. Each sub-section is stored as an
+``Interval`` object in ``interval_metadata``. Each ``Interval`` object
+has ``metadata`` keeping the information of this feature in the
+sub-section.
+
+To normalize the vocabulary between multiple formats (currently only
+the INSDC Feature Table and GFF3) to store metadata of interval
+features, we rename some terms in some formats to the same common name
+when parsing them into memory, as described in this table:
+
++-----------+-----------+-----------+---------+------------------------------+
+|INSDC      |GFF3       |Key stored |Value    |Description                   |
+|feature    |columns or |           |type     |                              |
+|table      |attributes |           |stored   |                              |
++===========+===========+===========+=========+==============================+
+|inference  |source     |source     |str      |the algorithm or experiment   |
+|           |(column 2) |           |         |used to generate this feature |
++-----------+-----------+-----------+---------+------------------------------+
+|feature key|type       |type       |str      |the type of the feature       |
+|           |(column 3) |           |         |                              |
++-----------+-----------+-----------+---------+------------------------------+
+|N/A        |score      |score      |float    |the score of the feature      |
+|           |(column 6) |           |         |                              |
++-----------+-----------+-----------+---------+------------------------------+
+|N/A        |strand     |strand     |str      |the strand of the feature. +  |
+|           |(column 7) |           |         |for positive strand, - for    |
+|           |           |           |         |minus strand, and . for       |
+|           |           |           |         |features that are not         |
+|           |           |           |         |stranded. In addition, ?  can |
+|           |           |           |         |be used for features whose    |
+|           |           |           |         |strandedness is relevant, but |
+|           |           |           |         |unknown.                      |
++-----------+-----------+-----------+---------+------------------------------+
+|codon_start|phase      |phase      |int (0,  |the offset at which the first |
+|           |(column 8) |           |1, or 2) |complete codon of a coding    |
+|           |           |           |         |feature can be found, relative|
+|           |           |           |         |to the first base of that     |
+|           |           |           |         |feature. It is 0, 1, or 2 in  |
+|           |           |           |         |GFF3 or 1, 2, or 3 in GenBank |
```

Comment:
I assume it is given the `Value type stored`, but the description has so far been used to describe the `Value type stored`.